### PR TITLE
check each variant for oversell and out of stock

### PR DESCRIPTION
**Pull request by tekhaus**
This task will now consider the oversell flag on a variant in addition to using the variant's **inventoryQuantity** instead of the **totalInventory** of the product. If at least one variant on a product is set to oversell, or has positive inventory, then the product will be considered in stock.

The task now also supports collection handles instead of titles for the optional inclusion and exclusion lists. The reason being that collection titles are not guaranteed to be unique.

PR for #112

### CODE REVIEW

File: docs/move-out-of-stock-products-to-the-end-of-a-collection/README.md [hunk 1]
```diff
@@ -27,8 +27,11 @@ This task re-sorts your collections, beginning with the sort order of your choic
...
-{% if options.run_hourly__boolean %}mechanic/scheduler/hourly{% endif %}
-{% if options.run_daily__boolean %}mechanic/scheduler/daily{% endif %}
+{% if options.run_hourly__boolean %}
+  mechanic/scheduler/hourly
+{% elsif options.run_daily__boolean %}
+  mechanic/scheduler/daily
+{% endif %}
 ```
 
 [Learn about event subscriptions in Mechanic](https://docs.usemechanic.com/article/408-subscriptions)
```
Comment (mattsodomsky) [inline docs/move-out-of-stock-products-to-the-end-of-a-collection/README.md:37]: https://learn.mechanic.dev/core/tasks/subscriptions

Update link 👆

**Review by mattsodomsky — CHANGES_REQUESTED** [review]
Fantastic, thank you.  One small docs change, and then I can merge in :)

File: docs/move-out-of-stock-products-to-the-end-of-a-collection/README.md (hunk 1 above)
Comment (tekhaus) [reply docs/move-out-of-stock-products-to-the-end-of-a-collection/README.md:37]: It looks like that link and a few others need to be updated in the task build readme template.

Comment (mattsodomsky) [reply docs/move-out-of-stock-products-to-the-end-of-a-collection/README.md:37]: Ahh :D indeed!

Comment (mattsodomsky) [reply docs/move-out-of-stock-products-to-the-end-of-a-collection/README.md:37]: Going to get this fix out there and then we'll get the auto-generated links fixed up

**Pull request merged**